nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -390,6 +390,15 @@
 
     <Control ID="RoundRobin"/>
 
+    <Control ID="Pause">
```

Comment:
Note: Add to the docs.nav2.org for new pages for these 2 nodes + migration guide update for the nodes and how to use them for pause / resume navigation behavior within the BT